src/session.rs

```diff
@@ -327,7 +401,7 @@ impl Session {
     ///
     /// # Ok(()) }
     /// ```
-    pub fn subsystem<S: AsRef<OsStr>>(&self, program: S) -> Command<'_> {
+    pub fn subsystem<S: AsRef<OsStr>>(&self, program: S) -> Command<&'_ Self> {
```

Comment:
Can we also have a generic version for subsystem please?

It cannot be emulated by to_command or to_raw_command

Reply:
Sure, should it have both the `to_` and the `arc` version, or just the `to_` version?

Reply:
Just to to version will be good enough.